examples/phi2/README.md

```diff
@@ -2,8 +2,19 @@
 
 ## Get the model
 
+Install the model builder script dependencies
```

Comment:
maybe put these into a requirements.txt file and refer to that instead so they don't need to be updated in multiple places.
e.g., src/python/models/export.requirements.txt